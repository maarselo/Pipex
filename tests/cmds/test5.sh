./pipex tests/infile/test5.txt "tail -n 2" "tr a-z A-Z" tests/output/test5.txt
