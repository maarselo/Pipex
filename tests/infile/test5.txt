línea 1
línea 2
línea 3

./pipex infile "tail -n 2" "tr a-z A-Z" outfile