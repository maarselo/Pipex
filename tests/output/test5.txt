LINEA 2
LINEA 3