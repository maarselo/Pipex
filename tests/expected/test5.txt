LÍNEA 2
LÍNEA 3